### DISCUSSION TITLE
🧐[问题]pro-form和antd的form有啥区别？有没有使用的文档？

### DISCUSSION THREAD

**Discussion by calmchang** [ant-design/pro-components#1236] [Q&A]
pro-form和antd的form有啥区别？有没有使用的文档？

**Comment by calmchang** [comment]
何为重型组件？什么场景下使用比较合适？如何确定使用pro-compoents组件还是使用antd原始组件？

**Comment by DerrickTel** [comment]
> pro-form和antd的form有啥区别？有没有使用的文档？

https://procomponents.ant.design/components/form

> 何为重型组件？什么场景下使用比较合适？如何确定使用pro-compoents组件还是使用antd原始组件？

重型组件：我说一下我个人的理解吧。我觉得是一个会依赖很多组件的组件，简单举个例子，比如说antd 的 Table 组件，他不单单是一个表格，附带了分页，筛选，选择等功能。
后面两个问题pro-compoents的官网有说明。
https://procomponents.ant.design/components/

**Comment by ByYogi** [comment]
pro组件封装很死，可修改的属性很有限。
如果遇到一个小问题，根本无从修复，社区又不大，最终只能到这来提问。最后要看源代码（也能学到很多）。
因为改项目维护人不多，活跃低，经常有一些莫名其妙的bug。
对于较大型项目来说，后期维护心智负担很大。

相对于pro，antd稳定性极高。bug少，就算有问题，在github社区反馈也很快。因为封装的少，都是一些必要的封装，所以用的时候掌控性很高，维护也方便。

**Comment by tunshiyu** [comment]
proform用处不是特别大，不过属于中间的工程，能指望什么直接的改善呢。下探到schema form 和 edittable之类的用起来确实有提效。
